fpdf/fpdf.py

```diff
@@ -1369,20 +1369,49 @@ def regular_polygon(self, x, y, numSides, polyWidth, rotateDegrees=0, style=None
         centerY = y - radius
         # center point is (centerX, centerY)
         points = []
-        i = 1
         for i in range(1, numSides + 1):
             point = centerX + radius * math.cos(
                 math.radians((360 / numSides) * i) + math.radians(rotateDegrees)
             ), centerY + radius * math.sin(
                 math.radians((360 / numSides) * i) + math.radians(rotateDegrees)
             )
             points.append(point)
-            i += 1
         # creates list of touples containing cordinate points of vertices
 
         self.polygon(points, style=style)
         # passes points through polygon function
 
+    @check_page
+    def star(self, x, y, r_in, r_out, corners, rotate_degrees=0, style=None):
+        """
+        Outputs a regular star with n corners.
+        It can be rotated.
+        It can be drawn (border only), filled (with no border) or both.
+
+        Args:
+            x (float): Abscissa of star's centre.
+            y (float): Ordinate of star's centre.
+            r_in (float): radius of internal circle.
+            r_out (float): radius of external circle.
+            corners (int): number of star's corners.
+            rotateDegrees (float): Optional degree amount to rotate star clockwise.
```

Comment:
The argument name needs to match the new `snake_case`

Reply:
Right! I missed it